README.md

```diff
@@ -0,0 +1,22 @@
+server: the DB + "manager"
+==========================
+
+Communicates with the controllers.
+
+Setup
+-----
+
+**Required Python version: 3.4**
+
+1. Create virtualenv if necessary: `pyvenv ./venv` (or `pyvenv-3.4 ./venv`)
+2. Activate virtualenv:
+   - bash, zsh: `source venv/bin/activate`
+   - fish: `. venv/bin/activate.fish`
+   - csh, tcsh: `source venv/bin/activate.csh`
+3. Install dependencies if necessary: `pip install -r requirements.txt`
+4. Configure by setting environment variables: `PORT` and `DB_URL`.  
+   You can also write it to `.env`, `./manage.py run` will load it.
+   `cp .env{.example,}; $EDITOR .env` to get started.
+5. add controller entries to the DB: `./manage.py controller MAC IP`
```

Comment:
Nerozumiem, co tento prikaz robi a co to znamena. (Mozno malo viem o gate...)
A tiez sa cudujem ze v prikaze nikde nie je "add". Mozno `addcontroller` apod? `controller add MAC IP`? (Zavisi ako si napr predstavujes ze sa (casom) budu mazat.)